### PR TITLE
add improvements from packages.git

### PR DESCRIPTION
**Pull request by aparcar**

### CODE REVIEW

File: docker-common.sh [hunk 1]
```diff
@@ -10,30 +10,35 @@ export USIGNHOME="/keys/usign/"
...
+
+# OpenWrt Build System (PGP key for 19.07 release builds)
+curl 'https://git.openwrt.org/?p=keyring.git;a=blob_plain;f=gpg/2074BE7A.asc' | gpg --import \
+ && gpg --fingerprint --with-colons '<pgpsign-19.07@openwrt.org>' | grep '^fpr:::::::::D9C6901F45C9B86858687DFF28A39BC32074BE7A:$' \
+ && echo 'D9C6901F45C9B86858687DFF28A39BC32074BE7A:6:' | gpg --import-ownertrust
 
-# Public usign key for unattended snapshot builds
-curl 'https://git.openwrt.org/?p=keyring.git;a=blob_plain;f=usign/b5043e70f9a75cde' \
-       	--create-dirs -o "$USIGNHOME/b5043e70f9a75cde"
+# untrusted comment: Public usign key for unattended snapshot builds
```
Comment (ynezz) [inline docker-common.sh]: Is this change needed?

File: docker-common.sh [hunk 2]
```diff
@@ -10,30 +10,35 @@ export USIGNHOME="/keys/usign/"
...
+curl 'https://git.openwrt.org/?p=keyring.git;a=blob_plain;f=usign/b5043e70f9a75cde' --create-dirs -o "$USIGNHOME/b5043e70f9a75cde" \
+ && echo "d7ac10f9ed1b38033855f3d27c9327d558444fca804c685b17d9dcfb0648228f *$USIGNHOME/b5043e70f9a75cde" | sha256sum -c
 
-# Public usign key for 19.07 release builds
-curl 'https://git.openwrt.org/?p=keyring.git;a=blob_plain;f=usign/f94b9dd6febac963' \
-	--create-dirs -o "$USIGNHOME/f94b9dd6febac963"
+# untrusted comment: Public usign key for 19.07 release builds
```
Comment (ynezz) [inline docker-common.sh]: Ditto.

**Comment by ynezz** [comment]
Apart from those nitpicks LGTM.

File: docker-common.sh (hunk 1 above)
Comment (aparcar) [reply docker-common.sh]: Following packages.git I guess

File: docker-common.sh (hunk 2 above)
Comment (aparcar) [reply docker-common.sh]: Ditto.
